ibis_substrait/compiler/translate.py

```diff
@@ -1369,6 +1369,11 @@ def _not_exists_subquery(
     )
 
 
+if not IBIS_GTE_71:
+    # NotExistsSubquery was removed in Ibis 7.1
+    translate.register(ops.NotExistsSubquery)(_not_exists_subquery)  # type: ignore[no-untyped-call]
```

Comment:
`NotExistsSubquery` was dropped [here](https://github.com/ibis-project/ibis/pull/7344).

Reply:
Can move the `_not_exists_subquery` definition under the `if not IBIS_GTE_71` condition but wanted to avoid the whitespace diff.